src/exp2.jl

```diff
@@ -3,9 +3,17 @@ for i=1:13
     include("exp_generated/exp_$i.jl")
 end
 
-function getmem(cache,k)
+function getmem(cache,k) # Called from generated code
     return cache[k-1];
 end
+function ldiv_for_generated!(C,A,B) # C=A\B. Called from generated code
```

Comment:
why not just use `ldiv!`?

Reply:
I think `ldiv!` is not intended for matrices but only factorizations. From `?ldiv!`:
```
  ldiv!(A, B)

  Compute A \ B in-place and overwriting B to store the result.

  The argument A should not be a matrix. Rather, instead of matrices it should be a
```

Reply:
just do the `lu!` first?

Reply:
Not sure exactly how `lu!`+`ldiv!` is different from `LAPACK.gesv!`. The `gesv!` does an LU and ldiv in fortran code, but maybe it does something more. They do have very small differences in CPU-timing and allocations. Both `_exp!` and `Base.exp` use the LAPACK-call.  

Would `lu!`+`ldiv!` make things easier in the CUDA-setting?

Reply:
That is the generic Julia linear algebra functions which, by default on dense matrices, calls LAPACK, but on other matrices (like CuMatrix) it then dispatches to something else.

Reply:
:+1: